resource/sites/dicmusic.com/config.json

```diff
@@ -80,6 +80,9 @@
       "privilege": "拥有无限邀请；佩戴6枚印记；创建6个私人合集；访问「Guru」版块；查看种子检查日志"
     }
   ],
+  "formerHosts": [
```

Comment:
`cdn` is missing, which should be

```
"cdn": ["https://dicmusic.club/", "https://dicmusic.com/"],
"formerHosts": [
    "dicmusic.club"
  ],
```
By the way, `formerHosts` shoud before `levelRequirements`, because `levelRequirements` can be very long.

Reply:
Appreciate for the suggestion ,I've updated it accordingly!